packages/functions/package.json

```diff
@@ -14,17 +16,28 @@
     "url": "https://github.com/vercel/vercel/issues"
   },
   "devDependencies": {
-    "vitest": "1.3.1"
+    "@aws-sdk/client-s3": "3.590.0",
+    "@aws-sdk/credential-provider-web-identity": "3.587.0",
+    "@smithy/types": "3.0.0",
+    "@types/jest": "29.5.12",
+    "jest-junit": "16.0.0",
+    "typescript": "5.4.5"
+  },
+  "peerDependencies": {
+    "@aws-sdk/credential-provider-web-identity": "3.x"
+  },
+  "peerDependenciesMeta": {
+    "@aws-sdk/credential-provider-web-identity": {
+      "optional": true
+    }
   },
   "engines": {
     "node": ">= 16"
   },
-  "files": [
-    "index.d.ts",
-    "index.js"
-  ],
   "scripts": {
-    "test": "vitest"
+    "build": "node ../../utils/build.mjs",
+    "test": "jest --reporters=default --reporters=jest-junit --env node --verbose --runInBand --bail",
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏We're moving tests over to `vitest`. Can we revert this change?

Reply:
Switch back to `vitest` in https://github.com/vercel/vercel/pull/11701/commits/1b9c4e261833e53a9af85d74a6a590099b18bb1a